src/test/java/com/mcmoddev/bot/helpers/forge/ForgeVersionHelperTest.java

```diff
@@ -0,0 +1,38 @@
+package com.mcmoddev.bot.helpers.forge;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ForgeVersionHelperTest {
+
+    @Test
+    void parseVersion() {
+        VersionMeta v = ForgeVersionHelper.getMCVersion("1.7.10-latest");
+        assertEquals(v.state, "latest");
+        assertEquals(v.version, "1.7.10");
+    }
+
+    @Test
+    void readVersions() {
+        try {
+            Map<String, ForgeVersion> result = ForgeVersionHelper.getForgeVersions();
+
+            assertTrue(result.size() > 0);
+            assertTrue(result.containsKey("1.7.10"));
+            assertEquals(result.get("1.7.10").getLatest(), "10.13.2.1343");
```

Comment:
According to the forge site, latest is 1614, error, or API wierdness?

Reply:
Looks like I missed a special case. The API is messed up.

Result:
![image](https://user-images.githubusercontent.com/25696312/73582300-8fe33900-445a-11ea-86da-40a597f1ce0b.png)

the second `1.7.10-latest-1.7.10` is overriding it :man_facepalming:

Reply:
Fixed in 8d287355bddfeffe0d633789fb3446b3f665c2ce, good catch!